_docs/developer/getting_started/commit_to_PR_from_fork.md

```diff
@@ -20,8 +20,14 @@ merging to the main branch.
 
 The instructions below are for command line use of git.
 
+1. **Before** you begin, confirm that your lcoal main branch of Submitty is
+    up-to-date. Additionally, if the PR's branch is not up-to-date with Submitty's main,
+    there will be a button on its Github webpage that you can press to update it.
```

Comment:
Github webpage -> page on Github (kind of personal preference, but I think it sounds better

Reply:
does "button on its page on Github" sound odd?